### PR TITLE
Test for addcomment added and minor tests for latest comment

### PR DESCRIPTION
**Pull request by robinzetterstrom**
* Tests for addComment added
* LatestComment.spec.js is added but not done yet

Test coverage is now at 100% but test cases have to be approved and added for LatestComment.spec.js

### CODE REVIEW

File: src/components/Modal.vue [hunk 1]
```diff
@@ -58,30 +58,32 @@
...
+        if (payload.username.length > 2 && payload.comment.length > 3 && payload.comment.length <= 300) {
+          // We pass in our form input and dispatch to store action 'add' and pass our payload to the store
+          this.$store.dispatch('add', payload)
           // ... after submit comment, reset form fields
-        this.form.username = ''
-        this.form.comment = ''
+          this.form.username = ''
+          this.form.comment = ''
+        } else {
+          return new Error("Couldn't add comment to store. There could be a problem with name or comment length")
```
Comment (mundish) [inline src/components/Modal.vue:75]: might wanna use alert here? Since the user won't se the error in the console.

File: test/unit/specs/LatestComment.spec.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      console.log(state)
+      done()
+    })
+  })
+
+  it('count comments and be a number', () => {
+    LatestComment.computed.latestComment.call(vm);
+    let counter = LatestComment.methods.countComments.call(vm, vm.latestcomment)
+
+    expect(counter).not.to.be.NaN
```
Comment (mundish) [inline test/unit/specs/LatestComment.spec.js]: Should we also test if it is a number? Currently we just check to see if it's not NaN (could still be null, undefined etc)?

File: src/components/Modal.vue (hunk 1 above)
Comment (robinzetterstrom) [reply src/components/Modal.vue:75]: Thats the meaning of it! 😄 
Users always get so mad when they get errors so why not make the error silent 

Just a joke though, i'll fix 👍
